# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/events/EventNameList.h
# Commit: f8a2a169b296
# Full Hash: f8a2a169b2969d29e04733ae27a56459e5359973
# Author: Alexandru Marc <amarc@mozila.com>
# Date: 2025-05-16 09:19:19
# Regressor Bug: 1550462
# File Overlap Count: 1
# Description:
#   Revert "Bug 1550462 - part 3: Enable `pointerrawupdate` event r=smaug" for causing mochitest failures @ test_pointerrawupdate_event_count.html
#   
#   This reverts commit eb27445e0352d6b4df540adb40e21f72fc052e80.
#   
#   Revert "Bug 1550462 - part 2: Make `PresShell::HandleEvent` dispatch preceding `pointerrawupdate` event r=smaug,dom-core,edgar"
# ==============================================================================

diff -r 8d25773fc82f -r f8a2a169b296 dom/events/EventNameList.h
--- a/dom/events/EventNameList.h	Fri May 16 06:20:47 2025 +0000
+++ b/dom/events/EventNameList.h	Fri May 16 09:30:09 2025 +0300
@@ -225,8 +225,6 @@
 EVENT(pointerout, ePointerOut, EventNameType_All, ePointerEventClass)
 EVENT(pointerenter, ePointerEnter, EventNameType_All, ePointerEventClass)
 EVENT(pointerleave, ePointerLeave, EventNameType_All, ePointerEventClass)
-EVENT(pointerrawupdate, ePointerRawUpdate, EventNameType_All,
-      ePointerEventClass)
 EVENT(gotpointercapture, ePointerGotCapture, EventNameType_All,
       ePointerEventClass)
 EVENT(lostpointercapture, ePointerLostCapture, EventNameType_All,